src/smc-webapp/project.coffee

```diff
@@ -41,7 +41,7 @@ project_store   = require('./project_store')
 {alert_message} = require('./alerts')
 misc_page       = require('./misc_page')
 
-{redux}         = require('./smc-react')
+{redux, ReactDOM} = require('./smc-react')
```

Comment:
I think, `ReactDOM` isn't used here. I also think, that inside a .coffee file it shouldn't be.